### PR TITLE
GSLUX-744 Feature measurements

### PR DESCRIPTION
**Pull request by tkohr**
### JIRA issue

https://jira.camptocamp.com/browse/GSLUX-744

### Description

PR calculates and updates the measurements of a drawn/edited feature.

It also adds two dependencies:
- cypress-real-events: for better testing of openlayers actions on the map
- eslint-plugin-no-only-tests: to prevent forgetting `only`s in test suites

To-dos:
- [x] Get elevation for points
- [ ] Calculate radius and adapt length and area calculation for circle (once it is managed as a circle geometry) => depends on GSLUX-743 and will be done in #159

### CODE REVIEW

File: src/components/draw/feature-measurements-helper.ts [hunk 1]
```diff
@@ -9,3 +16,26 @@ export function downloadCsv(profileData: PointData[]) {
...
+export function getElevation(coordinate: Coordinate) {
+  const lonlat = transform(coordinate, MAP_CRS, ELEVATION_CRS)
+  return fetch(`${ELEVATION_URL}?lon=${lonlat[0]}&lat=${lonlat[1]}`).then(
+    async response => {
+      const resp = (await response.json()).dhm
+      let text = i18next.t('N/A', { ns: 'client' })
+      if (resp) {
+        text = `${Number(resp).toFixed(2)} m`
+      }
+      return { formattedElevation: text, rawElevation: resp }
```
Comment (AlitaBernachot) [inline src/components/draw/feature-measurements-helper.ts:29]: Is there a reason why you return an object with the formatted value and the raw value? Maybe this: `text = '${Number(resp).toFixed(2)} m'` should be in getFormattedElevation function?

**Review by AlitaBernachot — APPROVED** [review]
Working great thanks (did not test elevation for point because I tested the github pages, but code looks ok).

Maybe extract formmating (in feat.measurement helper or from measurements.utils) to have the code for formatting text in the same place?

File: src/components/draw/feature-measurements-helper.ts (hunk 1 above)
Comment (AlitaBernachot) [reply src/components/draw/feature-measurements-helper.ts:29]: also don't know if this is relevant but maube we could create a directive for formatting to m, m2 or km?

Comment (tkohr) [reply src/components/draw/feature-measurements-helper.ts]: Good point, thanks for the idea! I've added a `format-distance` directive for the formatting logic from the helper. I'll continue to adopt it for the formatting in the utils in #159.

**Pull request merged**